src/pages/style.css

```diff
@@ -92,15 +92,10 @@ and negative margin.
     outline: none;
   }
 
-  /*
-  Not styling ::before with position: absolute and -5px positions because that
-  does not work on links that span more than one line.
-  */
   &:focus-visible {
     box-shadow: var(--token-focus-ring-box-shadow);
-    margin: -5px;
     outline: none;
-    padding: 5px;
+    z-index: 1;
```

Comment:
Minor nit, but will `z-index` here also need a non-default / non-`static` `position` in order to do anything? eg should we add `position: relative` to `.focus-ring-from-box-shadow` to ensure this applies? (Or should we apply `z-index` in the specific instances where it felt needed rather than for all `.focus-ring-from-box-shadow`?)

Reply:
Good call, I think I'll move `z-index`. We have some `absolute` positioned stuff that will eventually use this class (or some version of it) so setting `position: relative` will probably be too heavy of an assumption to make. I'll make this change as well!

Reply:
Updated in: https://github.com/hashicorp/dev-portal/pull/104/commits/89a70fb34e5c240c23182986fc78f570fc047457